zds/utils/api/permissions.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import permissions
+
+
+class UpdatePotentialSpamPermission(permissions.BasePermission):
+    """
+    Global permission check for blocked IPs.
```

Comment:
Cette description ne me parait pas adéquate. Je ne vois pas pourquoi ça parle d'IPs bloquées.

Reply:
J'avoue que je n'ai aucune idée de pourquoi cette description.
```suggestion
    Permission to use the API to mark a comment as (non)spam.
```